MockKitUI/MockPumpManager+UI.swift

```diff
@@ -105,3 +105,37 @@ extension MockPumpManager {
         return false
     }
 }
+
+public enum MockPumpStatusBadge: DeviceStatusBadge {
```

Comment:
[nit] Why the `MockPumpStatusBadge` if not used in `PumpStatusIndicator.pumpStatusBadge` below?

Reply:
Ah, thanks for reminding me. Was going to add some rudimentary timezone support to MockPumpManager here. Will do that.